### ISSUE TITLE
[StarLite Mk V] Missing wake from suspend option

### ISSUE THREAD

**Issue by alexreid** [StarLabsLtd/firmware#206]
Just upgraded my StarLite Mk V to firmware 24.07. The release notes say:

```
Add an option to disable waking from suspend by opening the lid
```

I've searched high and low in the firmware settings and haven't found this option yet – is this exposed in some other way?

**Comment by echo-devnull** [comment]
I've asked before as well: https://github.com/StarLabsLtd/firmware/issues/184#issuecomment-2258141190

@Sean-StarLabs can we turn it on now?   ;-)

**Comment by Sean-StarLabs** [comment]
Sure

**Comment by echo-devnull** [comment]
I do not understand? There is no such option in the bios right now, do you mean you are turning it on for 24.08 or something?

**Comment by Sean-StarLabs** [comment]
> do you mean you are turning it on for 24.08

Yes

**Comment by Sean-StarLabs** [comment]
Fixed with 00bd1d2ebfecc59f98d72892159bdee07ae12d36
